x-pack/plugins/security_solution/public/management/components/response_action_file_download_link/response_action_file_download_link.tsx

```diff
@@ -134,10 +134,10 @@ export const ResponseActionFileDownloadLink = memo<ResponseActionFileDownloadLin
     }, [action.isCompleted, action.wasSuccessful]);
 
     const downloadUrl: string = useMemo(() => {
-      return resolvePathVariables(ACTION_AGENT_FILE_DOWNLOAD_ROUTE, {
+      return `${resolvePathVariables(ACTION_AGENT_FILE_DOWNLOAD_ROUTE, {
         action_id: action.id,
         file_id: getFileDownloadId(action, agentId),
-      });
+      })}?apiVersion=2023-10-31`;
```

Comment:
have you considered keeping 2023-10-31 in a const to reuse it across security_solution?

Reply:
Hey @tomsonpl . I initially thought of that, but versions will diverge after the first release and thus I did not see it being a good approach to centralize the version level to `const`'s.